app/poros/item_poro.rb

```diff
@@ -0,0 +1,10 @@
+class ItemPoro
```

Comment:
This applies to `MerchantPoro` as well: these already live in a 'poros' directory and don't inherit from anything, so you could just name this class `Item`.